main.go

```diff
@@ -90,6 +92,10 @@ func main() {
 		"The region of the management cluster.")
 
 	// Monitoring configuration flags.
+	flag.BoolVar(&conf.Monitoring.AlertmanagerEnabled, "alertmanager-enabled", false,
+		"Enable Alertmanager controller.")
+	flag.StringVar(&conf.Monitoring.AlertmanagerSecretName, "alertmanager-secret-name", "",
```

Comment:
You know that is we were relying on labels instead of a secret-name here we could provide alerting to customers already? :D

Reply:
Let's not move too fast, having a name is fine for now. We can then think about what we want for customer later.

Reply:
Instead of getting the name here, we could list all secrets with a label of 
observability.giantswarm.io/tenant=anonymous and observability.giatnswarm.io/alerting=true

Reply:
I agree but I also don't want to make "rash" decisions that we cannot come out of but anyway I'm fine here